### ISSUE TITLE
Partitioning option

### ISSUE THREAD

**Issue by marchenko1985** [lnklnklnk/ga-bq#10]
Not sure why, but script does not create partitioned table

For small projects it may be ok, but if there is 1+ million records per day it may become not a cheap solution

So the question is: is it by design, or should we care at all about it?

May it be as an parameter in configuration or something like that

**Comment by ghost** [comment]
Could you explain to me what do u mean by 'script does not create partitioned table' in which part of the code is that? Maybe i can look into it. Im planning on using thin on a project that may have over 75M rows a day.

**Comment by ghost** [comment]
Reading a little more about this i find out that in order to set a partition time (assuming we are going to use day partitions) it should be done when the table is created.

**Comment by marchenko1985** [comment]
Yep, totally, all you need to do is to add following peace of code

![image](https://user-images.githubusercontent.com/88868/30981778-04ab0ad4-a48e-11e7-9c77-4b53b25d28c8.png)

and everything should work as expected, at least we were been able to reduce dramatically ammount of data being processed per request

but even with that still need some aggregation daily jobs with them we were able to reduce 20gb to 200kb data processed per DataStudio report

**Comment by ghost** [comment]
#18
